src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -164,7 +183,50 @@ public async Task<IEnumerable<User>> GetManyAsync(IEnumerable<Guid> ids)
                 new { Ids = ids.ToGuidIdArrayTVP() },
                 commandType: CommandType.StoredProcedure);
 
+            UnprotectData(results);
             return results.ToList();
         }
     }
+
+    private void ProtectData(User user)
+    {
+        if (user == null)
+        {
+            return;
+        }
+
+        user.MasterPassword = user.MasterPassword == null ? null :
+            string.Concat("P_", _dataProtector.Protect(user.MasterPassword));
```

Comment:
❓ Why "P"?

Reply:
"Protected" 🤷🏻 . Just wanted to add something to identify that this value came from these operations.